package.json

```diff
@@ -24,13 +24,10 @@
     "Visualization"
   ],
   "engines": {
-    "node": "18.x",
-    "pnpm": ">=8.6.0",
-    "vscode": ">=1.47.0"
+    "node": "20.x",
+    "pnpm": "8.x",
+    "vscode": ">=1.86.0"
   },
-  "activationEvents": [
-    "onCustomEditor:h5web.viewer"
-  ],
```

Comment:
Inferred from the `contributes` entries below since a recent-ish version of VS Code. Now that I've bumped the min version, I can remove it.